src/lib/CookieDecoder.ts

```diff
@@ -47,6 +47,10 @@ export class CookieDecoder {
         let id: string = '';
         for (let i = 0; i < 6; i++) id += this.readByte(bytes).toString(16);
 
+        // Status byte
+        const statusByte = this.readByte(bytes);
+        const keyMissing = (statusByte & 1) === 1;
```

Comment:
I'd say, to show that this is a binary setting, the `& 1` should be `& 0b1`.

Reply:
Bit flags are now available as enum (only `KEY_MISSING` for now)